guides/source/engines.md

```diff
@@ -1462,7 +1462,11 @@ end
 
 ### Available Load Hooks
 
-These are the load hooks you can use in your own code. To hook into the initialization process of one of the following classes use the available hook.
+These are the load hooks you can use in your own code.
```

Comment:
Let's just merge two and change `classes` to constants. `i18n` isn't a class and we put in the same table.

Reply:
Done